docs/REST API/rest-api.md

```diff
@@ -3454,67 +3455,125 @@ Retrieves the aggregated host OS versions information.
 | platform            | string   | query | Filters the hosts to the specified platform |
 | os_name     | string | query | The name of the operating system to filter hosts by. `os_version` must also be specified with `os_name`                                                 |
 | os_version    | string | query | The version of the operating system to filter hosts by. `os_name` must also be specified with `os_version`                                                 |
```

Comment:
we also support `team_id`

Reply:
Are we leaving `os_name` and `os_version` filter on here or is it being replaced with `os_version_id`?

Reply:
I'll defer to @rachaelshaw, but if we remove them it'll be a breaking change